src/main/java/com/example/dr_app/controller/DoctorController.java

```diff
@@ -0,0 +1,28 @@
+package com.example.dr_app.controller;
+
+import com.example.dr_app.model.Doctor;
+import com.example.dr_app.service.DoctorService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class DoctorController {
+    @Autowired
```

Comment:
Also, Autowired can be changed to AllArgsConstructor for the class and create a spring configuration for the controller. We can discuss this if it's too complicated.